doc/mkdocs/scripts/check_structure.py

```diff
@@ -23,6 +23,7 @@ def check_structure():
         'Exceptions',
         'Complexity',
         'Possible implementation',
+        'Default definition',
```

Comment:
Can you add this script to the Makefile? And then maybe run it as part of another step.

Reply:
Will do - good point!